monkestation/code/modules/viking/viking_axes.dm

```diff
@@ -0,0 +1,93 @@
+// fear the øx
+// øx means axe for the uncultured
+
+
+
+/obj/item/melee/viking
+	icon = 'monkestation/icons/viking/viking_items.dmi'
+	lefthand_file = 'monkestation/icons/viking/axes_lefthand.dmi'
+	righthand_file = 'monkestation/icons/viking/axes_righthand.dmi'
+	worn_icon = 'monkestation/icons/viking/viking_armor.dmi'
+
+/obj/item/melee/viking/tenja
+	name = "boarding axe"
+	icon_state = "hand_axe_item"
+	lefthand_file = "hand_axe_inhand_L"
+	righthand_file = "hand_axe_inhand_R"
+	worn_icon_state = "hand_axe_worn"
+	desc = "A one handed axe used by vikings."
+	force = 20
+	throwforce = 45
+	embedding = 50
+	wound_bonus = 25
+	slot_flags = ITEM_SLOT_BELT
+	w_class = WEIGHT_CLASS_NORMAL
+
+/obj/item/melee/viking/godly_tenja
+	name = "Leviathan Axe"
+	icon_state = "hand_axe_frost_item"
+	lefthand_file = "hand_axe_frost_inhand_L"
+	righthand_file = "hand_axe_inhand_R"
+	worn_icon_state = "hand_axe_frost_worn"
+	desc = "An axe with no equal to its power"
+	force = 25
+	throwforce = 65
+	embedding = 75
+	sharpness = SHARP_EDGED
+	wound_bonus = 30
+	slot_flags = ITEM_SLOT_BELT
+	w_class = WEIGHT_CLASS_NORMAL
+
+/obj/item/melee/viking/godly_tenja/afterattack(atom/target, mob/user, proximity_flag, click_parameters)
+	. = ..()
+	if(iscarbon(target))
+		var/mob/living/carbon/carbon_target = target
+		carbon_target.reagents.add_reagent(/datum/reagent/consumable/frostoil, 4)
+		carbon_target.reagents.add_reagent(/datum/reagent/consumable/ice, 4)
+
+/obj/item/melee/viking/genja
+	name = "battle axe"
+	base_icon_state = "battle_axe_item0"
+	lefthand_file = "battle_axe0"
+	righthand_file = "battle_axe0"
```

Comment:
Don't change the inhand_file. This is making it look for a file called "battle_axe0" not the actual sprites within. It won't work if you do it like this.